Makefile

```diff
@@ -27,8 +28,6 @@ NPM		:= npm
 #
 DOC_FILES	 = index.restdown
 JS_FILES	:= $(shell find lib test bin -name '*.js')
-JSL_CONF_NODE	 = tools/jsl.node.conf
```

Comment:
We should maintain the same convention and explicitly define the eslint configs here.